### DISCUSSION TITLE
Different Appnonce between irecovery -q and Blobsaver

### DISCUSSION THREAD

**Discussion by dying4004** [airsquared/blobsaver#392] [Q&A] [answered]
Hello

I just got a new iPhone 11 pro and was trying to get Appnonce for it.

Installed irecovery on my mac and got the appnonce from it and then ran blobsaver and got appnonce for the phone from blobsaver and these are different.

why is that and which one is the real one?

thanks

**Comment by airsquared** [comment, marked as answer]
They are different because your APNonce wasn't frozen and changed by itself. However, when you read with blobsaver for the first time it will freeze your APNonce so it doesn't change in the future.
